adamsjr8576/Refactor-Tractor
# Ja scss

What's this PR do?
 - This PR converts the styles.css file to implement sass creating a new styles, variables, mixins and normalize files. Refactor css for DRY code and increase general font-size and color contrast to increase accessibility. Also, refactor HTML for DRY code and eliminate unused classes and add id's for jQuery selecting instead of classes.  
 - added a svg for the friends card. 
 - utlizes scss variables and @extend
 - changes blue to darker blue and adds more yellow for styling

Where should the reviewer start?
 - The viewer should start by looking at the new styles.scss file, which contains the majority of the changes - There is a good bit of reference to the variables.scss file as well - then look at HTML to check classes and id's to see how they connect to script.js file.

How should this be manually tested?
- You could pull down and make sure that all looks good on your side.

Any background context you want to provide?
 - I've gotten good feedback on past projects that it is best practice to use classes for styling and id's for js selecting, which is why I made that decision and it also makes the HTML drier. The accessibility score is up to 91 with these changes! 

What are the relevant tickets? #6 -  #3 - #2 

Questions:

 - I think we should still split up the activity card into a couple smaller ones for easier use?

- Does this add new dependencies which need to be added?
 - I checked and it looks like scss is already added as a dependency so we should be all good there.

Screenshot: 
![screencapture-localhost-8080-2019-10-20-21_09_50](https://user-images.githubusercontent.com/49846853/67174108-242f0180-f37e-11e9-9c8e-c384a908c5dd.png)
